draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.scss

```diff
@@ -19,21 +17,47 @@ $dt-tooltip-content-border: $color-gray-300;
 
 .tooltipContent {
   border-radius: $border-solid-border-radius;
-  transition: opacity $ca-duration-fast, transform $ca-duration-fast;
-  padding: $ca-grid / 2 $ca-grid / 2;
-  border: $border-solid-border-width $border-solid-border-style
-    $dt-tooltip-content-border;
+  transition: opacity $animation-duration-fast,
+    transform $animation-duration-fast;
+  padding: 9px 9px;
```

Comment:
Is there a reason why you aren't using tokens for this anymore? Also, generally margin/padding is probably recommended to be in rem :)

Reply:
The $ca-grid tokens are deprecated so I've removed the component from using them.  The padding for the updated design was 9px and the spacing tokens do not have that in rem. I have updated it to be in rem rather than px but in this particular example there was no token that could be used.